domains/mc.notanicepenguin.json

```diff
@@ -0,0 +1,13 @@
+{
+  "owner": {
+    "username": "notanicepenguin"
+  },
+  "record": {
+    "SRV": {
+      "_minecraft._tcp": {
+        "port": 26009,
+        "target": "notanicepenguin.aternos.me"
+      }
+    }
+  }
```

Comment:
This is not the proper way to do SRV records. See docs: https://docs.is-a.dev/domain-structure/